bitnami/mongodb/templates/standalone/svc.yaml

```
@@ -35,6 +35,9 @@ spec:
   {{- if .Values.service.loadBalancerSourceRanges }}
   loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") }}
```

Little suggestion, `and` is not needed here.
```suggestion
  {{- if (eq .Values.service.type "LoadBalancer") }}
```

---

oh sure. missed that one. Thanks for the review :-D